src/Tribe/Editor/Blocks/Event_Datetime.php

```diff
@@ -50,4 +50,81 @@ public function assets() {
 			)
 		);
 	}
+
+	/**
+	 * Returns the block data for the block editor
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function block_data() {
+		$block_data = [
+			'id'         => $this->slug(),
+			'attributes' => [
+				'start'         => [
+					'type'   => 'string',
+					'source' => 'meta',
+					'meta'   => '_EventStartDate',
+				],
+				'end'           => [
+					'type'   => 'string',
+					'source' => 'meta',
+					'meta'   => '_EventEndDate',
+				],
+				'allDay'        => [
+					'type'   => 'boolean',
+					'source' => 'meta',
+					'meta'   => '_EventAllDay',
+				],
+				'timeZone'      => [
+					'type'   => 'string',
+					'source' => 'meta',
+					'meta'   => '_EventTimezone',
+				],
+				'separatorDate' => [
+					'type'   => 'string',
+					'source' => 'meta',
+					'meta'   => '_EventDateTimeSeparator',
+				],
+				'separatorTime' => [
+					'type'   => 'string',
+					'source' => 'meta',
+					'meta'   => '_EventTimeRangeSeparator',
+				],
+				'showTimeZone'  => [
+					'type'    => 'boolean',
+					'default' => tribe_get_option( 'tribe_events_timezones_show_zone', false ),
+				],
+				'timeZoneLabel' => [
+					'type'    => 'string',
+					'default' => class_exists( 'Tribe__Timezones' ) ? Tribe__Timezones::wp_timezone_string() : get_option( 'timezone_string', 'UTC' ),
+				],
+				// Only available for classic users
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 5b6e375ed3c0cc105a85a7bf749cd9a2</sub>